server/src/user/model.rs

```diff
@@ -100,7 +100,7 @@ impl User {
         })
     }
 
-    pub async fn add_role_mutation<'a, E>(
+    pub async fn add_role<'a, E>(
```

Comment:
When you change the name of the function you also need to change all calls of this functions (usually there is a refactoring command in vscode to do this -> see https://code.visualstudio.com/docs/editor/refactoring ).